app/views/checklist_tests/_checklist_measure.html.erb

```diff
@@ -13,20 +13,28 @@
   <div class = 'panel panel-default'>
     <div class = 'panel-heading'>
       <h1 class='panel-title'>
+       <% if !hide_patient_calculation?%>
         <%= "#{measure.cms_id} #{measure.name}" %>
+        <button class='btn btn-default' type='button' id='modifyrecord'>Edit Test</button>
+        <%else%>
+        <%= "#{measure.cms_id} #{measure.name}" %>
+        <%end%>
```

Comment:
Should the `else` and `end` have consistent indentation with the `if`? Also, should there be no space between the `<%` and `if`?